cmd/openshift-install/main.go

```diff
@@ -68,26 +71,20 @@ func main() {
 		for _, asset := range targetAssets {
 			st, err := assetStore.Fetch(asset)
 			if err != nil {
-				log.Fatalf("failed to generate asset: %v", err)
-				os.Exit(1)
+				logrus.Fatalf("Failed to generate asset: %v", err)
 			}
 
 			if err := st.PersistToFile(*dirFlag); err != nil {
-				log.Fatalf("failed to write target to disk: %v", err)
-				os.Exit(1)
+				logrus.Fatalf("Failed to write asset (%s) to disk: %v", asset.Name(), err)
 			}
 		}
 	case destroyCommand.FullCommand():
-		destroyer, err := destroy.New(l, *dirFlag)
+		destroyer, err := destroy.New(logrus.NewEntry(logrus.StandardLogger()), *dirFlag)
```

Comment:
It looks like this can just be `destroy.New(logrus.StandardLogger(), *dirFlag)` (at least that compiles for me ;).  Did you need `NewEntry` for something?

Reply:
Nope. I forgot Go did implicit type coercion on function parameters.